### ISSUE TITLE
Look into using the snapshot plugin of OAV

### ISSUE THREAD

**Issue by evalott100** [DiamondLightSource/python-artemis#299]
Currently we get a snapshot from the ffmpeg stream. There is  already a [plugin](https://areadetector.github.io/master/ADCore/NDFileJPEG.html) for taking snapshots on the oav which we could possibly use instead.

**Comment by DominicOram** [comment]
Needs discussion with @thomascobb and Neil

**Comment by DominicOram** [comment]
I don't think this is completed, we're still getting the snapshot from the stream so a discussion of the pros/cons of switching to the plugin is warranted

**Comment by dperl-dls** [comment]
oh, we should clarify wording then seeing as mjpg is also a plugin.
